docs/lit/examples/2_Plots.jl

```diff
@@ -18,44 +18,83 @@ using Plots, Unitful, UnitfulRecipes
 
 # ## Lines
 
-plot(Plots.fakedata(50, 5) * u"m", w=3)
+plot(Plots.fakedata(50, 5) * u"m", w = 3)
 
 # ## Parametric plots
 
-plot(t -> sin(t)*u"s", t -> sin(2t)*u"m", 0, 2π, line=4, leg=false, fill=(0, :orange))
+plot(
```

Comment:
Same as above, this turns a one-liner into an 9-lines example... IMHO it was better before 🤷‍♂️